meinberlin/apps/plans/templates/meinberlin_plans/plan_detail.html

```diff
@@ -145,8 +145,8 @@ <h3 class="item-detail-2__info-box-headline">{% trans 'Level of participation' %
                         {% else %}
                             {{ object.participation_explanation }}
                         {% endif %}
```

Comment:
This whole block needs to be indented one indent less. I.e. has to be shifted one indent left.